### ISSUE TITLE
unable to add user to project using UI

### ISSUE THREAD

**Issue by rajujith** [apache/cloudstack#9774]
##### ISSUE TYPE

 * Bug Report
 

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.1
~~~

##### CONFIGURATION

Project

##### SUMMARY

As a normal user, I can add a project and add a user to a project from another account in the same domain, I can't do the same with UI. UI checks the username with listusers which is not allowed for normal users for listing users in other accounts. 

##### STEPS TO REPRODUCE

~~~
1. Create a project as a normal user
2. add a user to the project filling the form with the exact username of another user.

~~~

##### EXPECTED RESULTS

~~~
UI should allow adding users to project since its allowed via API. 
~~~

##### ACTUAL RESULTS

~~~
UI doesn't allow adding user to project for normal users. 
~~~

**Comment by DaanHoogland** [comment]
@rajujith , I am not sure if allowing users to list users from another account would be a security issue. Maybe you can play with roles to see if it is possible?

**Comment by bernardodemarco** [comment]
Closing this ticket since it was fixed by #9927.
